src/main/kotlin/com/terraformation/backend/accelerator/DeliverableService.kt

```diff
@@ -21,23 +21,36 @@ class DeliverableService(
     private val submissionStore: SubmissionStore,
     private val systemUser: SystemUser,
 ) {
-  fun completeDeliverable(deliverableId: DeliverableId, projectId: ProjectId): SubmissionId {
+  fun completeDeliverable(
```

Comment:
Maybe rename this to `setDeliverableCompletion` or something like that, to reflect that it no longer just completes deliverables.